src/VR.js

```diff
@@ -610,37 +610,54 @@ class FakeVRDisplay extends VRDisplay {
 const createVRDisplay = () => new FakeVRDisplay();
 
 const controllerIDs = {
-  oculusVRIDLeft: 'Oculus Touch (Left)',
-  oculusVRIDRight: 'Oculus Touch (Right)',
-  oculusMobileVRIDLeft: 'Oculus Touch (Left)',
-  oculusMobileVRIDRight: 'Oculus Touch (Right)',
-  openVRID: 'OpenVR Gamepad',
-  openVRTrackerID: 'OpenVR Tracker'
+  OculusVRLeft: 'Oculus Touch (Left)',
+  OculusVRRight: 'Oculus Touch (Right)',
+  OculusQuestLeft: 'Oculus Touch (Left)',
+  OculusQuestRight: 'Oculus Touch (Right)',
+  OculusGo: 'Oculus Go',
+  GearVR: 'Gear VR',
+  OpenVR: 'OpenVR Gamepad',
+  OpenVRTrackerID: 'Tracker'
 };
 
+function getControllerID(vrDisplay, hand) {
+  if (!vrDisplay) { return; }
+  if (vrDisplay.id === 'OpenVR' ||
+      vrDisplay.id === 'GearVR' ||
+      vrDisplay.id === 'OculusGo') {
+    return controllerIDs[vrDisplay.id];
+  }
+  return controllerIDs[vrDisplay.id + hand.charAt(0).toUpperCase() + hand.slice(1)];
+}
+
 let gamepads = null;
 function getGamepads(window) {
   const {oculusVRDisplay, openVRDisplay, oculusMobileVrDisplay, magicLeapARDisplay} = window[symbols.mrDisplaysSymbol];
-  if (
+  const presentingDisplay =
     GlobalContext.fakeVrDisplayEnabled ||
```

Comment:
It wouldn't be the `presentingDisplay` in the fake vr display case based on this line. We should probably point to the actual fake VR display in the case it's detected.

Reply:
Done. Not sure if correct. not sure how to test that code path.

Reply:
For posterity, it would be running `examples/realitytabs.html`, in WebXR mode.